pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java

```diff
@@ -98,9 +101,9 @@ public InputStream getStream() throws IOException {
 
   @Override
   public void close() throws IOException {
-    StreamWrapperFinalizeAction finalizeAction = this.finalizeAction;
+    StreamWrapperCleaningAction finalizeAction = this.finalizeAction;
     if (finalizeAction != null) {
-      finalizeAction.close();
+      finalizeAction.onClean(false);
```

Comment:
Should calls like `.onClean(false)` be `finalizeAction.clean()` instead?

Reply:
So just change the name so that it is clearer ? Makes sense to me.